lib/pengine/bundle.c

```diff
@@ -1423,10 +1422,11 @@ pe__bundle_html(pcmk__output_t *out, va_list args)
              */
             uint32_t new_show_opts = show_opts | pcmk_show_implicit_rscs;
 
-            PCMK__OUTPUT_LIST_HEADER(out, FALSE, rc, "Container bundle%s: %s [%s]%s%s",
+            PCMK__OUTPUT_LIST_HEADER(out, FALSE, rc, "Container bundle%s: %s [%s]%s%s%s",
                                      (bundle_data->nreplicas > 1)? " set" : "",
                                      rsc->id, bundle_data->image,
                                      pcmk_is_set(rsc->flags, pe_rsc_unique) ? " (unique)" : "",
+                                     pcmk__s(desc, ""),
```

Comment:
Let's use the same parentheses style as the clone functions later in this commit